dev-docs/bidders/yieldlab.md

```diff
@@ -10,6 +10,7 @@ userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrable
 prebid_member: true
 pbjs: true
 pbs: true
+floors_supported: true
```

Comment:
thanks for adding this one -- can you also add all the other pieces of missing metadata as defined in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter ?

```
pbs_app_supported: true/false
fpd_supported: true/false
ortb_blocking_supported: true/partial/false
multiformat_supported: will-bid-on-one/will-bid-on-any/will-not-bid
deals_supported: true/false
gvl_id: NUMBER
coppa_supported: true/false
usp_supported: true/false
```

Reply:
Thank you for the catch we updated the missing pieces